duckplayer/duckplayer-impl/src/main/res/layout/activity_duck_player_settings.xml

```diff
@@ -125,6 +125,13 @@
                 app:primaryText="Open YouTube Videos in Duck Player"
                 />
 
+            <com.duckduckgo.common.ui.view.listitem.OneLineListItem
+                android:id="@+id/openDuckPlayerInNewTabToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="Open Duck Player in a new tab whenever possible"
```

Comment:
Will this be translated?

Reply:
Just extracted it to donottranslate and will now open a translations PR